contents/blog/customer-churn-analysis-guide.md

```diff
@@ -0,0 +1,296 @@
+
+## Understanding retention rate
+
+So that's churn rate, but what about retention rate and why, as I claimed earlier, is it a more useful metric? (most of the time) 
+
+To calculate your overall customer retention over a period you need to know:
+
+- How many customers you have at the start of a period (A)
+- How many customers you have at the end of a period (B)
+- How many new customers were added during that period. (C)
+
+You can then calculate your retention as a percentage (R) using the following formula:
+
+> ((B - C) / A) * 100 = R
+
+For example, if you have 100 customers at the start of a year, gain 35 new customers during the year and end the year with a total of 80 customers then your retention rate would be:
+
+> ((80-35)/100)*100 = 45% retention
+
+Meaning that each year a little more than half of your customers are churning and your usage as a whole is in decline. This would be a worrying sign!
+
+As you'll know if you didn't skip the intro, retention is just the inverse of churn rate. So, the being the case, why is retention rate **more useful** than churn rate?
+
+Because you can analyze retention using a cohort retention table like this:
+
+![new user retention](../images/blog/retention-vs-churn/retention-table-example-all.png)
+
+I'll explain how to read retention tables in a moment, but here's the rub... retention tables allow you to answer questions like:
+
+- Which features do users come back to over and over?
+- Which features do users try and never use again?
+- What actions contribute to users retaining or churning?
+- What properties do retaining users have that or churning users lack?
+
+If you can answer these questions, you're on the way the improving retention and reducing churn. And that's why retention rate is more useful, most of the time.
+
+### How to read (and use) a cohort retention table
+
+Trust me, it's less complicated than it looks.
+
+![new user retention](../images/blog/retention-vs-churn/retention-table-example-all.png)
+
+Above is a PostHog retention table for an imaginary Dropbox clone called Hedgebox. In this retention table we're looking at weekly repeat usage,  configured as:
+
+> `Unique users` who `signed up for the first time` in the last 8 weeks who then came back and triggered the `interacted with file` event in PostHog.
+
+To read it, just start on the left and work your way across:
+
+- **Cohort** Denotes the week these users signed up – i.e. the cohort these users belong to.
+
+- **Size:** Shows the size of that cohort – a useful reference to avoid comparing a very small cohort to a much larger one.
+
+- **Week 0:** Gives you the percentage of users in the cohort who used Hedgebox in that week. As it's week 0, this number will always be 100%.
+
+- **Week 1-8:** Shows the percentage of users in that cohort who returned to the product in each subsequent week – white boxes indicate periods that 'still in progress', blue boxes are completed periods.
```

Comment:
To clarify - the percentage is of the previous box, not the original box on the far left, correct?

Reply:
Or is it the other way round - it's worth over-explaining this!

Reply:
> To clarify - the percentage is of the previous box, not the original box on the far left, correct?

It's the percentage of the original box on the far left. I'm considering whether visualizing this might be better.